org-notebook.el

```diff
@@ -164,6 +164,8 @@
 						     ".png"
 						     )
 				))))
+    (or (f-directory-p "./img") (make-directory "./img"))
```

Comment:
Also move this line to in between lines 140 and 141, otherwise this autocreate img directory is useless.